### PR TITLE
Update CI.md

### PR DESCRIPTION
**Pull request by alinabuzachis**
Update CI.md

### CODE REVIEW

File: CI.md [hunk 1]
```diff
@@ -1,11 +1,11 @@
...
+## cloud.gcp_ops Collection
 
 GitHub Actions are used to run the Continuous Integration for redhat-cop/cloud.gcp_ops collection. The workflows used for the CI can be found [here](https://github.com/redhat-cop/cloud.gcp_ops/tree/main/.github/workflows). These workflows include jobs to run the sanity tests, linters and changelog check. The following table lists the python and ansible versions against which these jobs are run.
 
 | Jobs | Description | Python Versions | Ansible Versions |
 | ------ |-------| ------ | -----------|
 | changelog |Checks for the presence of Changelog fragments | 3.9 | devel |
-| Linters | Runs `ansible-lint`, `black`, `flake8`, and `isort` on plugins and tests | 3.9, 3.10 (ansible-lint), 3.11, 3.12 | devel |
-| Sanity | Runs ansible sanity checks | 3.9, 3.10, 3.11, 3.12 | 2.14 (not on py 3.12), 2.15 (not on py 3.12), 2.16 (not on py 3.9), Stable (not on py 3.9), Devel (not on py 3.9) |
+| Linters | Runs `ansible-lint`, `black`,`flake8` and `isort`| 3.9 and 3.11 for ansible-lint| devel |
```
Comment (mandar242) [inline CI.md:10]:
```suggestion
| Linters | Runs `ansible-lint`, `black`, `flake8` and `isort`| 3.9 and 3.11 for ansible-lint| devel |
```

File: CI.md [hunk 2]
```diff
@@ -1,11 +1,11 @@
...
+| Sanity | Runs ansible sanity checks | 3.9, 3.10, 3.11, 3.12 | 2.14 (not on3.12), 2.15 (not on 3.12), 2.16 (not on 3.9), stable (not on 3.9), Devel (not on 3.9) |
```
Comment (mandar242) [inline CI.md:11]:
```suggestion
| Sanity | Runs ansible sanity checks | 3.9, 3.10, 3.11, 3.12 | 2.14 (not on 3.12), 2.15 (not on 3.12), 2.16 (not on 3.9), stable (not on 3.9), Devel (not on 3.9) |
```

**Review by mandar242 — APPROVED** [review]
other than minor suggestions, LGTM!

File: CI.md [hunk 3]
```diff
@@ -1,11 +1,11 @@
...
 
 GitHub Actions are used to run the Continuous Integration for redhat-cop/cloud.gcp_ops collection. The workflows used for the CI can be found [here](https://github.com/redhat-cop/cloud.gcp_ops/tree/main/.github/workflows). These workflows include jobs to run the sanity tests, linters and changelog check. The following table lists the python and ansible versions against which these jobs are run.
 
 | Jobs | Description | Python Versions | Ansible Versions |
 | ------ |-------| ------ | -----------|
 | changelog |Checks for the presence of Changelog fragments | 3.9 | devel |
-| Linters | Runs `ansible-lint`, `black`, `flake8`, and `isort` on plugins and tests | 3.9, 3.10 (ansible-lint), 3.11, 3.12 | devel |
-| Sanity | Runs ansible sanity checks | 3.9, 3.10, 3.11, 3.12 | 2.14 (not on py 3.12), 2.15 (not on py 3.12), 2.16 (not on py 3.9), Stable (not on py 3.9), Devel (not on py 3.9) |
+| Linters | Runs `ansible-lint`, `black`, `flake8` and `isort`| 3.9 and 3.11 for ansible-lint| devel |
+| Sanity | Runs ansible sanity checks | 3.9, 3.10, 3.11, 3.12 | 2.14 (not on 3.12), 2.15 (not on 3.12), 2.16 (not on 3.9), stable (not on 3.9), devel (not on 3.9) |
```
[the remark this answers is not in the document]
Comment (alinabuzachis) [reply CI.md:11]: It's redundant because those requirements fall in the Python versions column.

**Review by GomathiselviS — APPROVED** [review]

**Pull request merged**